src/clients/apiReportClient/apiReportClient.ts

```diff
@@ -0,0 +1,26 @@
+import axios from 'axios';
+
+// import { store } from '../../domain/app/store';
+// import { apiTokenSelector } from '../../domain/auth/selectors';
+
+const axiosClient = axios.create({
+  baseURL: `${process.env.REACT_APP_API_REPORT_URI}`,
+  timeout: 30000,
+});
+
+// axiosClient.interceptors.request.use(
+//   (config) => {
+//     const token = apiTokenSelector(store.getState());
+//     if (token) {
+//       config.headers.Authorization = `Bearer ${token}`;
+//     }
+//     return config;
+//   },
+//   (error) => Promise.reject(error)
+// );
+
+export enum ROUTES {
+  EVENTS_ENROLMENTS = '/palvelutarjotinevent/enrolments/csv/',
+}
+
+export default axiosClient;
```

Comment:
Would it make sense to export helper functions that use axiosClient? Rathen than passing ROUTE.EVENTS_ENROLMENTS in other file

Reply:
I would actually like to tie the routes more to the client. Url mappings should be one of the clients features. But this also depends a lot on what we decide to do with the hook approach. I thought we could have a for every request, but if we don't use hooks, a helper library that uses client's features would be an alternative.